src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/x4decoder/common/stream/controllingmetric/StreamControllingMetric.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2022 AVI-SPL, Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.avdevices.encoderdecoder.haivision.x4decoder.common.stream.controllingmetric;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+/**
+ * Set of decoder controlling metric keys
+ *
+ * @author Harry
+ * @since 1.0
+ */
+public enum StreamControllingMetric {
+
+	STREAM_NAME("StreamName"),
+	ENCAPSULATION("Protocol"),
+	MULTICAST_ADDRESS("MulticastAddress"),
+	SOURCE_ADDRESS("SourceAddress"),
+	DESTINATION_PORT("DestinationPort"),
+	REJECT_UNENCRYPTED_CALLERS("RejectUnencryptedCallers"),
+	TYPE("Type"),
+	SRT_TO_UDP_STREAM_CONVERSION("SrtToUdpStreamConversion"),
+	ADDRESS("Address"),
+	DECODER_ID("DecoderId"),
+	FEC_RTP("FecRtp"),
+	ID("Id"),
+	LATENCY("Latency"),
+	PASSPHRASE("Passphrase"),
+	PASSPHRASE_SET("PassphraseSet"),
+	PORT("Port"),
+	SOURCE_IP("SourceIp"),
+	SOURCE_PORT("SourcePort"),
+	SRT_MODE("SrtMode"),
+	SRT_TO_UDP("SrtToUdp"),
+	SRT_TO_UDP_ADDRESS("SrtToUdp_address"),
```

Comment:
If this is not using to send the command, I think we should use PascalCase here

Reply:
I fixed it and the relevant case